lib/ansible/modules/system/user.py

```diff
@@ -1534,8 +1534,12 @@ def _get_user_property(self, property):
                 else:
                     return None
 
-    def _get_next_uid(self):
-        '''Return the next available uid'''
+    def _get_next_uid(self, system=None):
+        '''
+        Return the next available uid. If system=True, then
+        uid should be below of 500, if possible.
+        '''
+        system = (system == True)
```

Comment:
I see. I did a little research and indeed this is not pep8 conform (see [here](http://docs.quantifiedcode.com/python-code-patterns/readability/comparison_to_true.html)). Thank you for pointing that out! I will correct it immediately!